vumi/middleware/base.py

```diff
@@ -29,12 +37,16 @@ class BaseMiddleware(object):
 
     If you are subclassing this class, you should not override
     :meth:`__init__`. Custom setup should be done in
-    :meth:`setup_middleware` instead.
+    :meth:`setup_middleware` instead. The config class can be overidden by
+    replacing the ``config_class`` class variable.
     """
+    config_class = BaseMiddlewareConfig
```

Comment:
Perhaps this should be ``CONFIG_CLASS`` for consistency with applications and transports?